x-pack/osquerybeat/magefile.go

```diff
@@ -98,11 +98,6 @@ func CrossBuild() error {
 		return err
 	}
 
-	if runtime.GOARCH != "amd64" {
```

Comment:
I'm not sure if this will work, IIUC, the crossbuild is not supported in `arm64`